src/main/java/com/palantir/gradle/versions/VersionsProps.java

```diff
@@ -56,9 +53,9 @@ public static VersionsProps loadFromFile(Path path) {
         FuzzyPatternResolver.Builder builder = FuzzyPatternResolver.builder();
         recommendations
                 .stringPropertyNames()
-                .forEach(name -> builder.putVersions(
-                        name.replaceAll("/", ":"),
-                        recommendations.getProperty(name).trim()));
+                .forEach(name -> builder.putVersions(name.replaceAll("/", ":"), recommendations
+                        .getProperty(name)
+                        .trim()));
```

Comment:
This is worse, but I think it might require an exponential approach to fix: try both splitting the last argument and every argument on its own line, choose the one that's beter on some metric like LOC or max indent